AEPAudience/Sources/AudienceState.swift

```diff
@@ -509,12 +510,6 @@ class AudienceState {
         return self.syncedVisitorIds
     }
 
-    /// Returns the customer event data from the AudienceState instance.
-    /// - Returns: A dictionary containing the customer event data present in the signalWithData event
-    func getCustomerEventData() -> [String: String] {
-        return self.customerEventData
```

Comment:
Should you also remove the variable declaration on line 52? 
https://github.com/adobe/aepsdk-audience-ios/pull/48/files#diff-a75efdd486e17fe5367876949d483fc9fae74f9b2974aa9997a02748d1a65019R52